tests/acceptance/addon-smoke-test-slow.js

```diff
@@ -99,6 +100,68 @@ if (!experiments.MODULE_UNIFICATION) {
       expect(result.code).to.eql(0);
     }));
 
+    it('works when ember-cli-build is inside tests/dummy', co.wrap(function *() {
+      yield copyFixtureFiles('addon/ember-cli-build-at-tests');
+
+      let packageJsonPath = path.join(addonRoot, 'package.json');
+      let packageJson = fs.readJsonSync(packageJsonPath);
+
+      packageJson.dependencies = packageJson.dependencies || {};
+      // add HTMLBars for templates (generators do this automatically when components/templates are added)
+      packageJson.dependencies['ember-cli-htmlbars'] = 'latest';
```

Comment:
Can't this be put into the fixtures?

Reply:
Tried it initially but started getting `node_modules is empty` errors. Since this is all handled by the acceptance test helpers, I looked around and found this in a few other places.